module4/homework/appendNewRecipe/AppendNewRecipe.cpp

```diff
@@ -0,0 +1,70 @@
+#include <algorithm>
+#include <fstream>
+#include <iterator>
+
+#include <iostream>
+
+#include "AppendNewRecipe.hpp"
+
+bool AppendNewRecipe(std::vector<std::string> steps,
+                     const std::list<std::string>& ingredients,
+                     const std::deque<std::pair<size_t, char>>& amount) {
+    std::fstream recipes("recipes.txt", recipes.out | recipes.app);
```

Comment:
I would consider making file path a const variable and putting it in easy to access place.

Reply:
it's used just one so is that really a need for cons variable for path

Reply:
Firstly constexpr would be even better since it's value you know before compiling even starts, so compilator can make some optimalizations.
Secondly It's more about learning good practices. When you would work in larger project if you put path with other const/constexpr(sometimes you cannot use constexpr and you should use const) it would easier to find and change later than searching for "magic value"(And which one is the correct one?).